xapian-letor/Makefile.am

```diff
@@ -98,7 +98,7 @@ dist_m4data_DATA = m4-macros/xapianletor.m4
 lib_LTLIBRARIES = libxapianletor.la
 libxapianletor_la_SOURCES = $(lib_src)
 libxapianletor_la_LDFLAGS = \
-	$(XAPIAN_LDFLAGS) $(NO_UNDEFINED) -version-info $(LIBRARY_VERSION_INFO)
+	$(XAPIANLETOR_LDFLAGS) $(NO_UNDEFINED) -version-info $(LIBRARY_VERSION_INFO)
```

Comment:
`XAPIAN_LDFLAGS` was wrong here, but nothing sets `XAPIANLETOR_LDFLAGS` currently.

In `xapian-core/configure.ac` `XAPIAN_LDFLAGS` gets `-Bsymbolic-functions` added if that's supported - I think we want the equivalent for xapian-letor (`-Bsymbolic-function` is a linker option which binds references from the shared library to global symbols defined by the same library at library link time, which is more efficient that the default of resolving them when the library is dynamically loaded at runtime - the downside is that then you can't intercept uses of a symbol in the library by defining it in the program, but for a C++ API that is really unlikely to be something you'd want to be able to do).

Reply:
fixed in a fixup commit.